src/pages/user/authority/AuthorityOrcidModal.tsx

```diff
@@ -31,42 +28,38 @@ const StyledNormalText = styled.div`
   margin-top: 1rem;
 `;
 
-const AuthorityOrcidModal: FC = () => {
-  const { t } = useTranslation('common');
-  const user = useSelector((store: RootStore) => store.user);
-  const { location } = useHistory();
-
-  const noOrcid = user.authority?.orcids === undefined || user.authority?.orcids.length === 0;
-  const noAuthority = !user.authority;
-  const onHomePage = location.pathname === '/';
+interface AuthorityOrcidModalProps {
+  authority: Authority | null;
+}
 
-  const [hasClickedNext, setHasClickedNext] = useState(false);
+const AuthorityOrcidModal: FC<AuthorityOrcidModalProps> = ({ authority }) => {
+  const { t } = useTranslation('common');
   const [openOrcidModal, setOpenOrcidModal] = useState(false);
-  const [openAuthorityModal, setOpenAuthorityModal] = useState(noAuthority && onHomePage);
+  const [openAuthorityModal, setOpenAuthorityModal] = useState(true);
 
-  useEffect(() => {
-    setOpenOrcidModal(noOrcid && onHomePage && hasClickedNext);
-  }, [noOrcid, onHomePage, hasClickedNext]);
+  const noOrcid = !authority || !authority.orcids || authority.orcids.length === 0;
```

Comment:
@sarahse 
kan man koble til en autoritet som allerede har ORCID(s)? hvis ikke kan vi fjerne denne også

atm brukes den bare for å sjekke for om "next"-knappen skal rendres, hvis ikke skjer ingenting. Hvis man kan koble til en autoritet som allerede har ORCID burde vi vise en lukkeknapp elns

Reply:
nei, det skal ikke være mulig at en forfatter som ikke har en tilkoblet forfatterid har orcid. det finnes allerede et kryss på modalen da, og man kan klikke utenfor for å lukke den. det er ikke påkrevd å koble til orcid

Reply:
eller. når jeg tenker meg om, så er det jo flere som skriver til BARE, så kanskje man kan ha orcid der? @Eiriknil

Reply:
Skal vi bare fjerne sjekken da? Eller la den stå just in case?

Reply:
kanskje la den stå foreløpig